content/developer/reference/cmdline.rst

```diff
@@ -254,6 +254,11 @@ Emails
 
     Email address used as <FROM> when Odoo needs to send mails
 
+.. option:: --from-filter <address>
+
+    Define for which email address this SMTP configuration can be used (can be empty, a domain name or an entire email address).
```

Comment:
@std-odoo I've added the example below. The previous suggestion I commented with has an error in it. Can you review and update the RST documents?

``Define which email address the SMTP configuration will apply to. This field can be empty, a domain name, or an entire email address. If the sender's email address does not match this set filter, then the email will be encapsulated using a combination of the two system parameters: mail.default.from | mail.catchall.domain. For example "Admin" <admin@example.com> => "Admin" <notifications@mycompany.com>.``

Reply:
can't you make a PR to community ?

Reply:
@std-odoo I'm sorry, I'm not able to make a PR to community. I'm not aware of any documentation that could be updated in community. I can handle/file any dev and user facing documentation (published @ github.com/odoo/documentation). Are you referring to internal software notes in our backend to community? 

I think this PR is looking good. Are you able to make a change to the RST so we can move it to the next stage?

Reply:
Ah ok, didn't notice it was my 2 year old branch ^^

I update the rst